src/serialization.cpp

```diff
@@ -85,6 +86,10 @@ static Status serializePrecision(
     case ovms::Precision::U32:
     case ovms::Precision::U16:
     case ovms::Precision::U8:
+        if (servableOutput->getPostProcessingHint() == TensorInfo::ProcessingHint::STRING_2D_U8) {
```

Comment:
This code will be triggered for all types which is overhead.
Can we just set precision to BYTES before switch if processing hint is specified?